tests/test_quicksight.py

```diff
@@ -60,23 +62,78 @@ def test_quicksight(path, glue_database, glue_table):
         tags={"foo": "boo"},
     )
 
-    ingestion_id = wr.quicksight.create_ingestion("test-sql")
+    ingestion_id = wr.quicksight.create_ingestion(f"{resource_name}-sql")
     status = None
     while status not in ["FAILED", "COMPLETED", "CANCELLED"]:
-        status = wr.quicksight.describe_ingestion(ingestion_id, "test-sql")["IngestionStatus"]
+        status = wr.quicksight.describe_ingestion(ingestion_id, f"{resource_name}-sql")["IngestionStatus"]
     assert status == "COMPLETED"
 
-    ingestion_id = wr.quicksight.create_ingestion("test-sql")
-    wr.quicksight.cancel_ingestion(ingestion_id, "test-sql")
-    assert len(wr.quicksight.list_ingestions("test-sql")) == 3
+    ingestion_id = wr.quicksight.create_ingestion(f"{resource_name}-sql")
+    wr.quicksight.cancel_ingestion(ingestion_id, f"{resource_name}-sql")
+    assert len(wr.quicksight.list_ingestions(f"{resource_name}-sql")) == 3
 
     wr.quicksight.list_groups()
     wr.quicksight.list_iam_policy_assignments()
     wr.quicksight.list_iam_policy_assignments_for_user(wr.sts.get_current_identity_name())
     wr.quicksight.list_user_groups(wr.sts.get_current_identity_name())
     wr.quicksight.list_users()
-    wr.quicksight.get_dataset_ids("test-sql")
+    wr.quicksight.get_dataset_ids(f"{resource_name}-sql")
     wr.quicksight.get_data_source_ids("test")
 
-    wr.quicksight.delete_all_datasets()
-    wr.quicksight.delete_all_data_sources()
+    wr.quicksight.delete_all_datasets(regex_filter="test.*")
+    wr.quicksight.delete_all_data_sources(regex_filter="test.*")
+
+
+def test_quicksight_delete_all_datasources():
```

Comment:
This test is called `test_quicksight_delete_all_datasources` but it's still using a regex filer? It's a little confusing since the other test is called `test_quicksight_delete_all_datasources_filter`, but both of them are using a filter.

Reply:
Thanks, yea these are somewhat redundant so I've consolidated.